lib/LaTeXML/Core/Alignment.pm

```diff
@@ -583,9 +583,13 @@ sub normalize_sum_sizes {
     $colpos[$j] = Dimension($x);
     $x += $colwidths[$j];
     $x += $colrights[$j]; }
-  $$self{cwidth}       = Dimension($x);
-  $$self{cheight}      = Dimension($y);    # or account for vertical position of array as a whole?
-  $$self{cdepth}       = Dimension(0);
+
+  my $lastrow   = $rows[-1];
+  my $lastdepth = $$lastrow{cdepth} // Dimension(0);
+
+  $$self{cwidth} = Dimension($x);
+  $$self{cheight} = Dimension($y)->subtract($lastdepth); # or account for vertical position of array as a whole?
+  $$self{cdepth}  = $lastdepth;
```

Comment:
Now that the excitement of having align tikz-cd correctly is wearing off, changing the depth of the alignments is probably the wrong approach. The takeaway is that *TikZ matrices seem to align to the baseline of the last row*. How to best provide that baseline to TikZ (`@rowdepths`?!?) I just don't know: that depends on how this depth is used (or not used) elsewhere.

I also didn't notice 'bpadding' which would move the baseline further (maybe not important for TikZ, which probably has zero padding). Clearly this bit needs more thinking!

Reply:
I went for `@rowdepths`, so that other uses of `\halign` are not affected.